tests/functional/pages/surveys/dates/dates-answers.page.js

```diff
@@ -79,6 +79,42 @@ class DatesPage extends QuestionPage {
     browser.setValue('[name="non-mandatory-date-answer-year"]', year)
     return this
   }
+
+  setStartReportingPeriodDay(day) {
+    browser.setValue('[name="date-start-from-day"]', day)
+    return this
+  }
+
+  setStartReportingPeriodMonth(month) {
+    browser.selectByValue('[name="date-start-from-month"]', month)
+    return this
+  }
+
+  setStartReportingPeriodYear(year) {
+    browser.setValue('[name="date-start-from-year"]', year)
+    return this
+  }
+
+  setEndReportingPeriodDay(day) {
+    browser.setValue('[name="date-end-to-day"]', day)
+    return this
+  }
+
+  setEndReportingPeriodMonth(month) {
+    browser.selectByValue('[name="date-end-to-month"]', month)
+    return this
+  }
+
+  setEndReportingPeriodYear(year) {
+    browser.setValue('[name="date-end-to-year"]', year)
+    return this
+  }
+
+  setDatesAnswer(year) {
+    browser.setValue('[name="date-test-answer"]', year)
+    return this
+  }
+
```

Comment:
I don't think there should be any changes to anything in tests/functional in this pull request? (as the new conditional tests are done in tests/new_functional)